server.js

```diff
@@ -32,14 +33,6 @@ const server = new ApolloServer({
   context: async ({ req }) => {
     const { token } = req.headers || '';
 
-    logger.log({
```

Comment:
this is a logger for income request, it has to be here

Reply:
Put him on this place